### PR TITLE
Upgrade cf-deployment to 9.3

### PR DESCRIPTION
**Pull request by tlwr**
[Release notes](https://github.com/cloudfoundry/cf-deployment/releases/tag/v9.3.0)

What
----

Upgrade `cf-deployment` to 9.3 from 9.2

No major changes other than version bumps

Todo
----

- [x] Test with slim environment
- [ ] Test with HA environment

How to review
-------------

- Code review
- Look at pipelines

Who can review
--------------

Not @tlwr

### CODE REVIEW

File: manifests/cf-manifest/operations.d/329-capi-release.yml [hunk 1]
```diff
@@ -2,6 +2,6 @@
   path: /releases/name=capi
   value:
     name: "capi"
-    version: "1.81.0"
-    url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.81.0"
-    sha1: "8116229c5789dc47cc6ff35d59c8a5c7e363c174"
+    version: "1.82.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.82.0"
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/329-capi-release.yml]: Is this file required as it appears that [9.3 upstream is using CAPI 1.82](https://github.com/cloudfoundry/cf-deployment/blob/v9.3.0/cf-deployment.yml#L2341)?

File: manifests/cf-manifest/operations.d/229-cf-update_cflinuxfs_releases.yml [hunk 2]
```diff
@@ -3,6 +3,6 @@
   path: /releases/name=cflinuxfs3
   value:
     name: "cflinuxfs3"
-    version: "0.94.0"
-    url: "https://bosh.io/d/github.com/cloudfoundry/cflinuxfs3-release?v=0.94.0"
-    sha1: "c005fa270834b50eb1a5fb689cad8e6442f3cfbd"
+    version: "0.96.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry/cflinuxfs3-release?v=0.96.0"
```
Comment (LeePorte) [inline manifests/cf-manifest/operations.d/229-cf-update_cflinuxfs_releases.yml]: Is this override needed [upstream seems to have it](https://github.com/cloudfoundry/cf-deployment/blob/v9.3.0/cf-deployment.yml#L2357)?

File: manifests/cf-manifest/operations.d/329-capi-release.yml (hunk 1 above)
Comment (tlwr) [reply manifests/cf-manifest/operations.d/329-capi-release.yml]: Good shout - I've removed this opsfile

File: manifests/cf-manifest/operations.d/229-cf-update_cflinuxfs_releases.yml (hunk 2 above)
Comment (tlwr) [reply manifests/cf-manifest/operations.d/229-cf-update_cflinuxfs_releases.yml]: Good shout - I've removed this opsfile

**Comment by tlwr** [comment]
Closing, we have a story for cf-d 10
